src/main/java/org/cryptomator/common/mountpoint/CustomMountPointChooser.java

```diff
@@ -60,21 +70,47 @@ public boolean prepare(Volume caller, Path mountPoint) throws InvalidMountPointE
 				throw new InvalidMountPointException(new IllegalStateException("Not implemented"));
 			}
 		}
-		LOG.debug("Successfully checked custom mount point: {}", mountPoint);
-		return false;
```

Comment:
now, that all cases either throw or return, you might want to refactor this to

```java
return switch (caller.getMountPointRequirement()) {
    case FOO -> {
        // ...
        yield true;
    }
    // ...
}
```

Reply:
Fixed in [14dc026](https://github.com/cryptomator/cryptomator/pull/2082/commits/14dc026354730ede3866e2f167166d0e1bd63c09)